Dijkstra's Algorithm.java

```diff
@@ -0,0 +1,78 @@
+import java.util.*;
+public class Dijkstra {
```

Comment:
@Geeky-Ashu14 That's a very good start!

You can provide a space after the imports, so that the code is propery formatted.